polling_stations/apps/data_importers/management/commands/import_torridge.py

```diff
@@ -36,62 +50,46 @@ def address_record_to_dict(self, record):
-            "10003502901",  # FLAT 1, 39-41 WELL STREET, TORRINGTON
             "10003506177",  # THE COACH HOUSE, LITTLE SILVER, STEVENSTONE, TORRINGTON
             "10002301760",  # HIGHER COURT BARN, ST. GILES, TORRINGTON
             "10090288709",  # THE BEECHES, LIMERS LANE, NORTHAM, BIDEFORD
             "100040374230",  # THE COTTAGE, DIDDYWELL ROAD, NORTHAM, BIDEFORD
             "10003507823",  # STANLEIGH LODGE, DIDDYWELL ROAD, NORTHAM, BIDEFORD
             "10095917163",  # THE QUILLET CARAVAN, DIDDYWELL ROAD, APPLEDORE, BIDEFORD
             "10003506180",  # ORCHARD COTTAGE LANE TO WAY BARTON, ST GILES IN THE WOOD
-            "10003505437",  # THE VALE, DUNSLAND CROSS, BRANDIS CORNER, HOLSWORTHY
-            "10003505435",  # LYNE AKRES, DUNSLAND CROSS, BRANDIS CORNER, HOLSWORTHY
             "10003506037",  # THE SPINNEY, WINSFORD LANE, HALWILL JUNCTION, BEAWORTHY
-            "10002299135",  # CHURCH VIEW FARM, ASHWATER, BEAWORTHY
             "10091078138",  # WHEELERS RETREAT, BRADWORTHY, HOLSWORTHY
             "10002299506",  # FLAT LITTLE BURSDON ROAD PAST EAST BURSDON FARM, HARTLAND
             "10090289801",  # LITTLE BURSDON CARAVAN, HARTLAND, BIDEFORD
             "10002300463",  # LITTLE BURSDON, HARTLAND, BIDEFORD
             "10003506595",  # TREVENNA, HARTLAND, BIDEFORD
-            "10002298352",  # GOOSEDOWN, DOLTON, WINKLEIGH
+            "200001768435",  # SMUGGLERS COTTAGE, BUCKS MILLS, BIDEFORD
+            "10002300522",  # HIGH PARK LODGE, LITTLEHAM, BIDEFORD
+            "10090289146",  # APARTMENT 3436 WESTBEACH RESORT BATH HOTEL ROAD, WESTWARD HO!
+            "10002299752",  # COMMON MOOR, CLAWTON, HOLSWORTHY
+            "10093911404",  # MEADOW WOOD, FRITHELSTOCK, TORRINGTON
+            "10003506170",  # BITEFORD, WOOLSERY, BIDEFORD
         ]:
             return None
 
         if record.addressline6 in [
             # splits
-            "EX22 7XE",
-            "EX39 3LU",
             "EX38 8NN",
-            "EX39 3LX",
-            "EX21 5HN",
+            "EX39 3LU",
             "EX39 5FL",
             "EX39 1PU",
+            "EX39 3LX",
+            "EX21 5HN",
+            "EX22 7XE",
+            # suspect
             "EX39 3TP",  # BUCKLAND VIEW, BIDEFORD
-            "EX39 3NZ",  # NORTHDENE, BIDEFORD
             "EX39 1HA",  # BURROWS WAY, NORTHAM, BIDEFORD
             "EX39 1HB",  # SANDYMERE ROAD, NORTHAM, BIDEFORD
             "EX39 5ED",  # WALLAND DRIVE, BUCKS CROSS, BIDEFORD
             "EX38 8BJ",  # KITCHENER ROW, CALF STREET, TORRINGTON
+            "EX22 7NA",
+            "EX22 7YH",
         ]:
             return None
 
         return super().address_record_to_dict(record)
-
-    def station_record_to_dict(self, record):
-        # Newton St Petrock Community Centre, Newton St Petrock
-        if record.polling_place_id == "8386":
-            record = record._replace(polling_place_postcode="EX22 7LW")
```

Comment:
This station is missing a postcode in the council data, so this fix still seems useful

Reply:
Added it in! Also will stop moving the methods around.